aten/src/ATen/native/cuda/BinaryMiscBackwardOpsKernels.cu

```diff
@@ -8,22 +8,41 @@
 #include <ATen/native/DispatchStub.h>
 #include <ATen/native/TensorIterator.h>
 #include <ATen/native/cuda/Loops.cuh>
+#include <ATen/native/cuda/JitLoops.cuh>
 
 // NOTE: CUDA on Windows requires that the enclosing function
 // of a __device__ lambda not have internal linkage.
 
 namespace at {
 namespace native {
 
+const char sigmoid_backward_name[] = "sigmoid_backward_kernel";
 void sigmoid_backward_kernel_cuda(TensorIteratorBase& iter) {
-  if(isComplexType(iter.dtype())) {
-    AT_DISPATCH_COMPLEX_TYPES(iter.dtype(), "sigmoid_backward_cuda", [&]() {
+  auto dtype = iter.dtype();
+  if(isComplexType(dtype)) {
+#if AT_USE_JITERATOR()
+    static const auto sigmoid_backward_string = jiterator_stringify(
+        template <typename T>
+        T sigmoid_backward(T a, T b) {
```

Comment:
Since this is `sigmoid_backward`, you should update `const char sigmoid_backward_name[] = "sigmoid_backward_kernel";` -> `const char sigmoid_backward_name[] = "sigmoid_backward";`

Reply:
Done that :) Thanks, @kshitij12345!